### ISSUE TITLE
chrome browser 

### ISSUE THREAD

**Issue by t1m02k16** [pimatic/pimatic#1030]
Hello,
I use the chrome browser. It is not possible for me to change the "VariableInputDevice", "max! Plugin" etc
with the "+" and "-". 
With my iPhone and the Iexplorer it is no problem.

**Comment by mwittig** [comment]
Same here! See also

- https://forum.pimatic.org/topic/1789/how-to-use-variableinputdevice/8
- https://github.com/pimatic/pimatic/issues/626

**Comment by mwittig** [comment]
The following demo does not work for me either on Chrome http://dev.jtsage.com/jQM-Spinbox/ - maybe it is due to the following: http://stackoverflow.com/questions/40257614/jquery-click-event-on-select-element-not-working-as-desired-on-chrome-54

**Comment by thost96** [comment]
This works fine for my using Windows 10 and Chrome version 55.0.2883.87.

**Comment by mwittig** [comment]
Same platform and version I have, btw

**Comment by thost96** [comment]
@mwittig yes, this also works fine for me.

**Comment by t1m02k16** [comment]
Version 55.0.2883.87 m (64-bit)
-> dont work! http://dev.jtsage.com/jQM-Spinbox/

**Comment by mwittig** [comment]
@t1m02k16 I am also using the 64-bit version. @thost96 Maybe you have the 32-bit version?

**Comment by thost96** [comment]
@mwittig i'm also using the 64 bit version. What excatly does not work for you both?

**Comment by t1m02k16** [comment]
We are not able to controll with "+" and "-" 
If we press the Button, nothing happens

**Comment by thost96** [comment]
Ok, as i sayed earlier, works perfect for my.

**Comment by mwittig** [comment]
Same here. I even tried with a new user profile as I suspected some experimental functions settings (chrome://flags) may cause the issue. However, no joy.

**Comment by t1m02k16** [comment]
Now, i´m at work
-Windows 7 Prof. 32bit
-Chrome Version 55.0.2883.87 m
=Everything is working fine... No Problems! strange!!!!

**Comment by t1m02k16** [comment]
@mwittig are you using a Surface?

**Comment by mwittig** [comment]
No. It is a standard notebook. I think I have found the sollution following the advice given in the following forum thread (second last post): https://productforums.google.com/forum/#!topic/youtube/mYBzHWscubo
I used the malware removal tool from malwarebytes.com as suggested and found 8 adware things on my computer (not FlashMall  btw). After performing the removal and rebooting the notebook the demo works as expected.

**Comment by t1m02k16** [comment]
didnt work :( 
Malwarebytes found nothing...

**Comment by t1m02k16** [comment]
I tried everythind install/deinstall scan with 5 differernt antivir/antimal
-> i cant use the buttons :/
